com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/font/BMFont.java

```diff
@@ -366,7 +366,7 @@ public Boolean parseText(BufferedInputStream data) throws BMFontFormatException,
                 } break;
                 case "char":
                 {
-                    assert(this.chars <= this.charArray.size() + 1 );
+                    assert(this.chars >= this.charArray.size());
```

Comment:
The same issue with `assert`. This assert works in JUnit tests and fails build because condition always false. It is always has been like that, but `assert` turned off by default.